NuRadioMC/simulation/simulation.py

```diff
@@ -90,7 +92,7 @@ def __init__(self, inputfilename,
                  debug=False,
                  evt_time=datetime.datetime(2018, 1, 1),
                  config_file=None,
-                 log_level=logging.STATUS,
+                 log_level=LOGGING_STATUS,
```

Comment:
What would happen in several unit tests, is that `simulation.py` would be imported before the `module.setup_logger()` was called. The `logging.STATUS` attribute is only defined when that function is executed. On import, the class definition would be evaluated and the test would fail because at that point `logging.STATUS` is not defined yet. Therefore, instead of forcing the `setup_logger()` to always happen before any other import statements (which is difficult to read and remember), I opted to define the global variable which can be used for this purpose.

If you have a suggestion for how to solve this differently, let me know!

Reply:
I can't think of a better solution, so I'm fine with it ;-)